src/amberscript/parser_buffer_test.cc

```diff
@@ -762,5 +765,142 @@ INSTANTIATE_TEST_SUITE_P(
                     NameData{"mat2x2"},
                     NameData{"mat2x2<>"}));  // NOLINT(whitespace/parens)
 
+TEST_F(AmberScriptParserTest, BufferWithStructStd140) {
```

Comment:
Need some tests with vectors, arrays and runtime arrays.

Reply:
Added vec test. Array and runtime array will be followups when we add the array syntax.